frontend/src/pages/projects/screens/projects/ProjectView.tsx

```diff
@@ -1,23 +1,36 @@
 import * as React from 'react';
 import ApplicationsPage from '~/pages/ApplicationsPage';
+import { useAccessReview } from '~/api';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
 import EmptyProjects from './EmptyProjects';
 import useUserProjects from './useUserProjects';
 import ProjectListView from './ProjectListView';
 
+const accessReviewResource: AccessReviewResourceAttributes = {
+  group: 'project.openshift.io',
+  resource: 'projects',
+  verb: 'create',
+};
+
 const ProjectView: React.FC = () => {
   const [projects, loaded, loadError, refreshProjects] = useUserProjects();
+  const [allowCreate, rbacLoaded] = useAccessReview(accessReviewResource);
 
   return (
     <ApplicationsPage
       title="Data science projects"
       description="View your existing projects or create new projects."
```

Comment:
@kywalker-rh For the title of the page -- simple, just say what you can do?

Can you create a project?

- yes: "View your existing projects or create new projects."
- no: "View your existing projects." 
    - Might be weird? You will see an empty page for existing projects... but the other message on the empty state will fill in what to do

Reply:
I think this makes sense, if you don't have any existing projects, then there is nothing to see, so this description covers the possibility.